readme_template.md

```diff
@@ -1,6 +1,8 @@
 ----
-**Take Note!**  This version of the Hosts file generator, and tests, are for Python 3.5+ only.
-
+**Take Note!**
+- This version of the Hosts file generator, and tests, are for Python 3.5+ only.
+- This repository centers around the hosts file aggregation Python script and NOT around the hosts files it produces. Issues regarding the personal management of hosts files, such as hosts file managers, are out of scope. However, as a courtesy, there are some recommended hosts manager options available.
```

Comment:
I feel this is too strong, @ScriptTiger.  Of course we care about the hosts files we produce.

I also feel we're more about the *curation* of hosts sources, than the tool (the python code).  For every release, I read the diffs from sources.  That's more valuable than the Python code which, at core, combines and deduplicates hosts.

I also think, people rarely read the readme attentively.  The Readme is too long, already.  I also think our root folder is so crowded, people currently must scroll-scroll merely to get to the top of the readme.

Reply:
Maybe something along these lines would be better suited for an issue template, but for now just dropped it.